Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
```

Comment:
Should probably do this rebased from https://github.com/openshift/cluster-logging-operator/pull/736

Reply:
Done

Reply:
Done.